neuroinformatics/hbp_prov/hbp-prov-schema-v3.0.json

```diff
@@ -32,7 +32,8 @@
             "properties": {
                 "id": {
                     "type": "string",
-                    "format": "uuid"
+                    "description": "A Universally unique identifier (we only allow lowercase UUIDs).",
+                    "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
```

Comment:
can you extract pattern to the definitions and reference via $ref keyword like it was done in early versions? It's a pain to detect changes in these duplicated regexes.